opentelemetry/proto/common/v1/common.proto

```diff
@@ -53,3 +53,10 @@ message StringKeyValue {
   string key = 1;
   string value = 2;
 }
+
+// InstrumentationLibrary is a message representing the instrumentation library informations
```

Comment:
```suggestion
// InstrumentationLibrary is a message representing the instrumentation library information
```

You might want to reference the glossary at this point since people tend to confuse those terms.
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/glossary.md#instrumenting-library